app/javascript/Pages/components/Footer/footer.scss

```diff
@@ -0,0 +1,80 @@
+.footer {
+  background-color: #252636;
+  padding: 4rem 0 3rem 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-self: center;
+
+  .footer-logo {
+    display: flex;
+    justify-content: center;
+    margin: 0 auto 20px auto;
+  }
+
+  .social-text {
+    display: flex;
+    justify-content: center;
+    margin: 40px auto 0 auto;
+    color: #fff;
+    font-weight: 600;
+    font-size: 16px;
+  }
+
+  .social-media {
+    width: 100%;
+    max-width: 1000px;
+    display: flex;
+    justify-content: center;
+    margin: auto;
+
+    .social-media-wrapper {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      max-width: 1000px;
+      margin: 40px auto 0 auto;
+
+      .social-icons {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        width: 240px;
+
+        .social-icon-link {
```

Comment:
**issue (blocking)**: although SASS allows us to place classes inside each other, it's generally a bad practice to do it because this will make the class overly specific and will prefent reusability of styles.
When you do this the compiled css will be `.footer .social-media .social-media-wrapper .social-icons .social-icon-link` 🤯
If then you need to use the social icons in a sidebar or header the class will not work.
Generally, we only put classes inside each other when it's a modifier or temporary state, the "&" is replaced with the class you are inside of. e.g.:
```scss
.button {
    &.is-disabled {
        ...
    }
    .button-group & {
        ...
    }
    &:before {
        ...
    }
    @media screen and (max-width: 960px) { // this way you don't need to repeat the class name
        ...
    }
}
```
That is why we like to the BEM formatting, the name of the class should indicate where it should be used:
`.footer__link` should be placed inside `.footer`, `.button--primary` should be added to an element with `.button`

Reply:
ok I may not have understood SASS as well as I thought. I understand the disadvantages you explained but I still have doubts about the "&". If I have the class footer, and another class footer-link, I couldn't do .&-link, right?

Reply:
actually yes 🙈 inside `.footer` you could do `&-link` which would compile to `.footer-link`, but you shouldn't! In this case becase if you want to change the name "footer-link" to something else you can't do a "search and replace" easily (because in some places it will be "footer-link" and others "&-link", and if you have other classes ending with "&-link" you'd be renaming things you shouldn't).
TLDL: don't use "&" to compose class names, only to make the selector explicit (e.g. "&.is-disabled" shows the class is meant to be used in the same element, ".button-group &" means the class is applied to the parent)